### ISSUE TITLE
Inventory motherboard shown as not present [Witherspoon]

### ISSUE THREAD

**Issue by gkeishin** [openbmc/openbmc#1216]
```
  "data": {
    "/xyz/openbmc_project/inventory/system/chassis/motherboard": {
      "BuildDate": "", 
      "Manufacturer": "0000000000000000", 
      "Model": "", 
      "PartNumber": "0000000", 
      "Present": 0, 
      "PrettyName": "SYSTEM PLANAR   ", 
      "SerialNumber": "000000000000"
    }
```

Had a word with @dkodihal and opening this defect and he knows what to fix for this one.. Cheers !!!

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/5872> Inventory:- adding the extra properties for motherboard
Resolves  #1216 Inventory motherboard shown as not present [Witherspoon]

**Comment by ratagupt** [comment]
# **STATUS**
